packages/ui/bundlewatch.config.json

```diff
@@ -2,7 +2,7 @@
   "files": [
     { "path": "./dist/ui.browser.js", "maxSize": "19.5KB" },
     { "path": "./dist/framework*.js", "maxSize": "44KB" },
-    { "path": "./dist/vendors*.js", "maxSize": "43KB" },
+    { "path": "./dist/vendors*.js", "maxSize": "143KB" },
```

Comment:
Driveby-comment: This seems like an awfully big increase for what I expect will be a rarely used feature? 🤔 Or does this end up in a separate chunk that gets lazy-loaded somehow?